.babelrc

```diff
@@ -0,0 +1,20 @@
+{
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "node": "10"
+        }
+      }
+    ],
+    [
+      "@babel/preset-typescript"
+    ]
+  ],
+  "plugins": [
+    "@babel/proposal-class-properties",
+    "@babel/plugin-proposal-numeric-separator",
+    "@babel/plugin-proposal-optional-chaining"
```

Comment:
It doesn't seem like optional chaining is part of the current node LTS.

Reply:
Or a part of TypeScript 3.6, but should be a part of 3.7

Reply:
I will be part of the next typescript in some weeks, having the plugin just means that babel supports the transformation not that it will be used.